recipe/meta.yaml

```diff
@@ -16,16 +16,50 @@ build:
     - gradio = gradio.cli:cli
     - upload_theme = gradio.themes.upload_theme:main
   noarch: python
-  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
-  number: 0
+  number: 1
 
 requirements:
+  build:
+    - nodejs >=14
+    - pnpm =9
   host:
-    - python {{ python_min }}
+    - python >={{ python_min }},<3.13
     - hatchling
     - hatch-requirements-txt
     - hatch-fancy-pypi-readme >=22.5.0
     - pip
+    # copied from below
+    - python >={{ python_min }},<3.13
+    - aiofiles >=22.0,<25.0
+    - anyio >=3.0,<5.0
+    #- audioop-lts <1.0 Not yet avalaible yet and only work with python 313
+    - fastapi >=0.115.2,<1.0
+    - ffmpy
+    - groovy-python >=0.1,<0.2
+    - gradio-client ==1.13.0
+    - httpx >=0.24.1
+    - huggingface_hub >=0.28.1
+    - jinja2 <4.0
+    - markupsafe >=2.0,<4.0
+    - numpy >=1.0,<3.0
+    - orjson >=3.0,<4.0
+    - packaging
+    - pandas >=1.0,<3.0
+    - pillow >=8.0,<12.0
+    - pydantic >=2.0,<2.12
+    - python-multipart >=0.0.18
+    - pydub
+    - pyyaml >=5.0,<7.0
+    - ruff >=0.9.3
+    - safehttpx >=0.1.6,<0.2.0
+    - semantic_version >=2.0,<3.0
+    - starlette >=0.40.0,<1.0
+    - tomlkit >=0.12.0,<0.14.0
+    - typer >=0.12,<1.0
+    - typing-extensions ~=4.0
+    - urllib3 ~=2.0
+    - uvicorn >=0.14.0
```

Comment:
I wonder if it's possible to read these packages from requirements.txt so that they are automatically updated when we add a new requirement in gradio?